cob_monitoring/src/battery_monitor.py

```diff
@@ -152,53 +174,84 @@ def say(self, text):
             for component in self.sound_components:
                 sss.say(component, [text])
 
+    def trigger_horn(self):
+
+        if self.enable_horn != True:
+            raise ValueError(f"could not signal horn, not enabled")
```

Comment:
why do you raise a `ValueError` here? Wouln't it be sufficient to just return the function (maybe with some `rospy.logdebug`?

Reply:
Done!